articles/best-javascript-animation-libraries/index.md

```diff
@@ -0,0 +1,451 @@
+        const cube = new THREE.Mesh(geometry, material);
+        scene.add(cube);
+        function render(time) {
+          time *= 0.0002; // convert time to seconds
+          cube.rotation.x = time;
+          cube.rotation.y = time;
+          renderer.render(scene, camera);
+          requestAnimationFrame(render);
+        }
+        requestAnimationFrame(render);
+      }
+      main();
+    </script>
+  </body>
+</html>
+```
+style.css file:
+
+```css
+body {
+  margin: 0;
+  height: 200vh;
+}
+canvas {
+  display: block;
+}
+```
+### GreenSock.js
+
+[GreenSock Animation Platform (GSAP)](https://greensock.com/) is a powerful JavaScript animation library that is lightweight. It makes animations flow faster and easy to use. GSAP enables developers to create animations that look great in all major browsers, such as, Chrome, Firefox Mozilla, and Internet Explorer 11+.
+It has a ton of features that make it unique:
+GSAP is used with DOM elements, CSS animations, SVG, WebGL, strings, colors, and motion paths. Also, it solves problems when things fall apart on browsers, thus enabling animations to work well.
+Here is a simple example of GSAP:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+      body {
+        background: #f3034f;
+        display: flex;
+        width: 92vw;
+        height: 88vh;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .circle {
+        background: #78db74;
+        width: 50px;
+        height: 50px;
+        border-radius: 100%;
+      }
+    </style>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.5.1/gsap.min.js"></script>
+    <title>GreenSockJs Animation</title>
+  </head>
+  <body>
+    <div class="circle"></div>
+    <script>
+      gsap.to(".circle", {
+        duration: 3,
+        x: 1000,
+        scale: 4,
+      });
+    </script>
+  </body>
+</html>
+```
+### Popmotion.js
+
+[Popmotion.js](https://popmotion.io/) is a lightweight and robust JavaScript animation library. It is powerful because it supports animations like Keyframes, spring, decay, and complex animations.
+Popmotion.js makes it easy for front-end developers and web designers to animate 3D objects, React components, SVG, CSS, and DOM elements.
+
+### Typed.js
+
+[Typed.js](https://mattboldt.com/demos/typed-js/) is a JavaScript animation library that provides the experience of creating a typing animation on your webpage. It has various options to modify how the animations function.
+Typed.js options include; strings, type speed, start delay, loop, back speed, and back delay.
+
+To get started with type.js, download the latest version of the animation library from GitHub. Then include the script to your page using a script tag.
+```html
+ <script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.11"></script>
```

Comment:
You only need the CDN file: https://github.com/mattboldt/typed.js/blob/master/README.md